documentation/modules/managing/con-cluster-loss-recovery.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="cluster-loss_{context}"]
+= Recovery from loss of a Kafka cluster
+
+A Kafka cluster might be lost, for example, when using cloud computing services.
```

Comment:
I think this can easily happen even without a cloud. So I would maybe leave this sentence out.

Reply:
Removed.